topic_folders/hosts_instructors/resources_for_online_workshops.md

```diff
@@ -0,0 +1,51 @@
+s
+In the wake of COVID-19 in early 2020, The Carpentries community came together to share experiences, tips and best practices for teaching online. This page lists all the resources developed and links to ongoing conversations by The Carpentries Core Team and community on different platforms.
+
+#### Resources by The Carpentries
+
+The Carpentries convened a [Task Force to address the urgent demand for remote Carpentries workshops](https://github.com/carpentries/task-forces/blob/master/2020/COVID-19/COVID-19-charter.md) as communities have shifted to distance work across the globe. The Task Force concluded its work on April 1, with an initial set of guidelines for teaching, supporting, and communicating about fully remote versions of all Data Carpentry, Library Carpentry, and Software Carpentry workshops.
```

Comment:
We are calling it online workshops all over, should we not replace remote with online?